presto-spi/src/main/java/com/facebook/presto/spi/type/UnparametrizedDecimalType.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.facebook.presto.spi.type;
-
-import com.facebook.presto.spi.ConnectorSession;
-import com.facebook.presto.spi.block.Block;
-import com.facebook.presto.spi.block.BlockBuilder;
-
-/**
- * Dummy unparametrized Decimal type just to be used in function signatures before specialization.
- * Not in actual data processing.
- */
-public final class UnparametrizedDecimalType
```

Comment:
we probably should have dropped this class earlier. As I do not believe it was needed even before your patch.